mlrun/feature_store/api.py

```diff
@@ -355,12 +355,10 @@ def ingest(
         featureset.purge_targets(target_names=purge_target_names, silent=True)
     else:
         for target in purge_targets:
-            overwrite_supported_targets = [TargetTypes.parquet, TargetTypes.nosql]
-            if target.kind not in overwrite_supported_targets:
+            overwrite_not_supported_targets = [TargetTypes.csv]
```

Comment:
If someone will create a new target class, zero chances they'll remember to update here whether it supports overwrite or not, this should be an attribute of the class or something